app-drawer/app-drawer.html

```diff
@@ -108,24 +126,41 @@
     }
 
     #scrim {
-      position: fixed;
-      top: -120px;
+      position: absolute;
+      top: 0;
+      bottom: 0;
       left: 0;
       right: 0;
-      bottom: -120px;
       background: var(--app-drawer-scrim-background, rgba(0, 0, 0, 0.5));
       opacity: 0;
-      transition: opacity 0.2s ease;
+      visibility: hidden;
```

Comment:
Hmm... I commented out [L136](https://github.com/PolymerLabs/app-layout/blob/fixed-drawer-v2/app-drawer/app-drawer.html#L136) and [L159](https://github.com/PolymerLabs/app-layout/blob/fixed-drawer-v2/app-drawer/app-drawer.html#L159) and then ran demo/demo6.html, I couldn't see the flashes when toggling the layout.